CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
   - Include targetUrl in the output of the `service:check` command [#1072](https://github.com/apollographql/apollo-tooling/pull/1072)
   - Import apollo-env utility types directly instead of treating them as globals [#1074](https://github.com/apollographql/apollo-tooling/pull/1074)
   - Add `--markdown` output option to `service:check` [#1072](https://github.com/apollographql/apollo-tooling/pull/1072)
+  - Enhance formatting for `service:check` output [#1146](https://github.com/apollographql/apollo-tooling/pull/1146)
```

Comment:
This entry and the one above it (I believe) should be placed under the Upcoming header, rather than within an already published version of the changelog. Ping me if you're unsure!

Reply:
oh man what a silly mistake

Reply:
There's literally no documentation about how to update the changelog, so I don't think it's silly.